content/blog/2021-07-tonic-0-5.md

```diff
@@ -0,0 +1,174 @@
+---
+date: "2021-07-09"
+title: "Announcing Tonic 0.5"
+description: "July 9, 2021"
+---
+
+We are pleased to announce version 0.5 of [Tonic], a native gRPC implementation in
+Rust. 0.5 is a big release and has been in the works for some time.
+
+Some key new features are:
+
+# gRPC-Web
+
+[gRPC-Web] is a protocol that allows clients to connect to gRPC services over
+HTTP/1.1, instead of the usual HTTP/2. A common use-case for gRPC-Web is
+JavaScript clients running in the browser.
+
+The new crate [tonic-web] allows regular Tonic servers to accept gRPC-Web
+requests without the need of an external proxy.
```

Comment:
it may be worth mentioning something about the proxies here before saying they aren't necessary anymore. Maybe something like:

> Previously, Tonic only supported gRPC over HTTP/2, so serving RPCs from gRPC-Web clients required the use of a proxy to translate HTTP/2 to HTTP/1.1. Now, the new [`tonic-web`] crate allows regular Tonic servers to ...